src/main/java/org/folio/rest/impl/HoldingsTypeAPI.java

```diff
@@ -229,5 +210,31 @@ private void internalServerErrorDuringPut(Throwable e, String lang, Handler<Asyn
         .respond500WithTextPlain(messages.getMessage(lang, MessageConsts.InternalServerError))));
   }
 
+  private Future<String> saveHoldingsType(PostgresClient pgClient, HoldingsType entity) {
+    Future<String> future = Future.future();
+    pgClient.save(REFERENCE_TABLE, entity.getId(), entity, future);
+    return future;
+  }
 
+  private PostHoldingsTypesResponse handleSaveHoldingsTypeException(Throwable t) {
+    if (PgExceptionUtil.isUniqueViolation(t)) {
+      Error error = new Error()
+        .withCode("name.duplicate")
+        .withMessage("Cannot create entity with not unique name")
```

Comment:
Consider changing "not unique" -> "nonunique" or rephrasing e.g. "Cannot create entity; name is not unique"

Reply:
Also consider making this a static constant so it can be referenced in the unit tests and you don't have to worry about keeping the string literals synchronized